extension/lib/buildBoltCart.js

```diff
@@ -29,13 +41,13 @@ module.exports = async function buildBoltCart (context, input) {
     }
   })
 
-  const tax = input.totals.find(({ type }) => type === 'tax')
+  const tax = getTotal('tax', input.totals)
   const discounts = input.totals.filter(({ type }) => type === 'discount')
   const cart = {
     order_reference: input.cartId,
     currency: input.currency,
-    total_amount: input.totals.find(({ type }) => type === 'grandTotal').amount * 100,
-    tax_amount: tax ? tax.amount * 100 : undefined,
+    total_amount: getTotal('subTotal', input.totals) + tax,
+    tax,
```

Comment:
was the change from `tax_amount` to `tax` on purpose? the output is now different

Reply:
no it wasn't. good catch

Reply:
it is fixed now.